### PR TITLE
Fixes #31076 - hammer import  endpoint for cvv and library

### PR DESCRIPTION
**Pull request by parthaa**
This commit provides the bindings to import from a version to a view or
to the library.

### CODE REVIEW

**Comment by parthaa** [comment]
Related to https://github.com/Katello/katello/pull/9042

**Comment by parthaa** [comment]
Follow the steps listed here https://github.com/Katello/katello/pull/9042#issuecomment-729947436

**Comment by chris1984** [comment]
Starting to test

**Review by chris1984 — APPROVED** [review]
Looks amazing, thanks @parthaa

File: lib/hammer_cli_katello/content_view.rb [hunk 1]
```diff
@@ -106,10 +106,12 @@ class CreateCommand < HammerCLIKatello::CreateCommand
       failure_message _("Could not create the content view")
 
       option ["--composite"], :flag, _("Create a composite content view")
-
+      option ["--import-only"], :flag, _("Designate this Content View for "\
```
Comment (jturel) [inline lib/hammer_cli_katello/content_view.rb:109]: Does this undo the server-side mechanism we use to conditionally show or hide the parameter?

File: lib/hammer_cli_katello/content_import.rb [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      desc _('Imports archive to a content view version')
+      action :version
+      command_name "version"
+
+      include HammerCLIForemanTasks::Async
+      include ContentImportCommon
+    end
+
+    class LibraryCommand < HammerCLIKatello::SingleResourceCommand
+      desc _('Imports archive to the organization library lifecycle-environment')
```
Comment (jturel) [inline lib/hammer_cli_katello/content_import.rb:51]: suggestions

1. Imports a content archive to an organization's library
2. Imports a content archive to an organization's library lifecycle environment (no hyphen)

**Review by jturel — COMMENTED** [review]
LGTM. minor feedback

File: lib/hammer_cli_katello/content_view.rb (hunk 1 above)
Comment (parthaa) [reply lib/hammer_cli_katello/content_view.rb:109]: No, we do the same for composite as you see above. So trying to stay consistent with other boolean params

**Review by chris1984 — APPROVED** [review]
ACK, was able to get the export to show the path in quotes and metadata file generated during an export, and was able to import both library and version with the updated changed. Thanks @parthaa

**Pull request merged**